dapp/tests/management/management.spec.ts

```diff
@@ -23,4 +31,44 @@ test.describe.parallel('Name Management Tests', () => {
         sharedState.wallet.address = address;
         sharedState.wallet.mnemonic = mnemonic;
     });
+
+    test('Renew an owned name', async ({ appPage: page, context, sharedState }) => {
+        const keypair = Ed25519Keypair.deriveKeypair(sharedState.wallet.mnemonic ?? '');
+        const name = generateRandomName('display');
+        const { response } = await purchaseName(name, sharedState.wallet.address ?? '', keypair);
+        expect(response.effects?.status.status).toBe('success');
+
+        await page.goto('/my-names');
+        await expect(
+            page.getByTestId('name-card').filter({ hasText: normalizeIotaName(name, 'at') }),
+        ).toBeVisible({ timeout: 10_000 });
+
+        const nameCard = page
+            .getByTestId('name-card')
+            .filter({ hasText: normalizeIotaName(name, 'at') });
+
+        await nameCard.getByTestId('name-card-avatar').hover();
+        const menuButtonLocator = nameCard.getByTestId('menu-button');
+        await expect(menuButtonLocator).toBeVisible();
+        await menuButtonLocator.click();
+
+        await page.getByText('Renew Name', { exact: true }).click();
+        const dialog = page.getByRole('dialog');
+        await expect(dialog.getByText('Renew Name', { exact: true })).toBeVisible();
+
+        const yearSelect = dialog.getByRole('combobox');
+        if (await yearSelect.isVisible().catch(() => false)) {
+            await expect(yearSelect).toBeVisible();
+        }
```

Comment:
isn't this reduntant? if it fails it doesn't go in, but if it's tru it's because `isVisible()` returns true, why would you check it again `await expect(yearSelect).toBeVisible();`